docs/user-docs/auto-mode.md

```
@@ -22,6 +22,33 @@ Plan (with integrated research) → Execute (per task) → Complete → Reassess
+
+For projects that need more up-front discovery, enable deep planning mode in project preferences:
+
+```yaml
+planning_depth: deep
+```
+
+You can also opt in when starting project setup with `/gsd new-project --deep` or `/gsd new-milestone --deep`; GSD writes the project `.gsd/PREFERENCES.md` setting for you.
+
+Deep mode keeps the normal slice execution loop, but first runs a one-time staged discovery flow before milestone-level planning:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)